src/triage/component/catwalk/evaluation.py

```diff
@@ -110,72 +111,177 @@ def _validate_metrics(
                 raise ValueError("For custom metric {} greater_is_better must be "
                                  "boolean True or False".format(name))
 
-    def _generate_evaluations(
+    def _build_parameter_string(
+        self,
+        threshold_unit,
+        threshold_value,
+        parameter_combination
+    ):
+        """Encode the metric parameters and threshold into a short, human-parseable string
+
+        Examples are: '100_abs', '5_pct'
+
+        Args:
+            threshold_unit (string) the type of threshold, either 'percentile' or 'top_n'
+            threshold_value (int) the numeric threshold,
+            parameter_combination (dict) The non-threshold parameter keys and values used
+
+        Returns: (string) A short, human-parseable string
+        """
+        full_params = parameter_combination.copy()
+        if not (threshold_unit == 'percentile' and threshold_value == 100):
```

Comment:
This makes the code more generalizable, in that you can now send metrics without thresholds through _evaluations_for_threshold() with a threshold of 100_pct and it will simply print nothing. But it might be better if, when the user specifies 100_pct threshold (instead of listing no threshold), it prints out 100_pct. For example, I recently ran a query against an evaluations table with `WHERE parameter LIKE '%_pct'`. With this change, that wouldn't return what I might have expected it to return.